src/file-uploader/file-uploader.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface FileItem {
+	file: File;
+	state: string;
```

Comment:
This should have a union type like `"edit" | "upload" | "complete"` (or whatever the complete set is)